ChangeLog.md

```diff
@@ -26,6 +26,8 @@ Current Trunk
 - Added new linker option -s WASM=2 which produces a dual Wasm+JS build, which
   falls back to using a JavaScript version if WebAssembly is not supported in
   target browser/shell. (#10118)
+- Added new linker option -s CLOSURE_WARNINGS_ARE_ERRORS=1 that allows aborting
```

Comment:
I think it is better to have these be two separate things, so that there is more flexibility. Some projects might have hundreds of C/C++ compilation warnings, but would like to address Closure warnings. Or other projects might think Closure warnings are not as essential, but they'd like to address compiler warnings.

For example, there is no chance that I would be able to enable both -Werror and -s CLOSURE_WARNINGS_ARE_ERRORS=1 to our Unity build system, people would be on my throat before the day is over. But -s CLOSURE_WARNINGS_ARE_ERRORS=1 would be much easier to pass, as very few people work on the JS layer and there's much less JS code to deal with.

Reply:
Ah OK.   Then maybe we can use the `WarningManager` then?   Do we would be able to so `-Wclosure` or `-Werror -Wno-closure`.   

If you want to get this landed we can also do that as a followup?  I'm mostly trying to keep the number of settings from growing if I can.